MSAL/test/unit/native_auth/input_validator/MSALNativeAuthInputValidatorTest.swift

```diff
@@ -39,4 +39,15 @@ final class MSALNativeAuthInputValidatorTest: XCTestCase {
     func testInput_whenInvalidStringInputIsUsed_resultShouldBeInvalid() {
         XCTAssertFalse(validator.isInputValid(""))
     }
+    
+    func testInput_whenValidArrayInputIsUsed_resultShouldBeValid() {
+        XCTAssertTrue(validator.isInputValid(["email@contoso.com"]))
+        XCTAssertTrue(validator.isInputValid(["email", "password"]))
+        XCTAssertTrue(validator.isInputValid([1, 2, 3]))
+        XCTAssertTrue(validator.isInputValid([""]))
```

Comment:
One question, do we want `[""]` to be considered a valid input from the user?

Reply:
Good question. The method is generic, it only takes into account the existence of items in the array, not its contents. That's why I tested it even with numbers. I can change it to receive an array of strings, and also check if the items are empty strings or not (this validation is correct for scopes, since MSAL ignores empty strings in the list). In this case the method won't be generic, but we can create new methods in the future if we need to validate other input types. What do you think @diegojerezba ?

Reply:
I think it's fine the way it is now, thanks for the clarification 👍